docs/static/includes/interfaces/bicep/int.sg.udt.schema.bicep

```
@@ -0,0 +1,19 @@
+// ============== //
+//   Parameters   //
+// ============== //
+
+import { serviceGroupTargetResourceIdsType } from 'br/public:avm/utl/types/avm-common-types:>version<'
+@description('Optional. The resource IDs of the service groups you wish to associate this resource with.')
+param serviceGroupTargetResourceIds serviceGroupTargetResourceIdsType[]?
```

Similar to the comment I added to the types PR itself: What speaks against

```suggestion
@description('Optional. The resource IDs of the service groups you wish to associate this resource with.')
param serviceGroupTargetResourceIds string[]?
```